server/room.ts

```diff
@@ -65,6 +66,8 @@ export class Room {
     if (roomData) {
       this.deserialize(roomData);
     }
+    // check if room has a currently subscribed owner and set room capacity
+    this.checkSub();
```

Comment:
I would rename this to just "isRoomOwnedByCurrentSubscriber" and then do the side effect directly in this constructor